content/ui-patterns/empty-states.mdx

```diff
@@ -0,0 +1,58 @@
+
+### Primary Text
+Use primary text to explain the purpose of the empty state, help users feel comfortable to engage with the content, or begin a feature flow. Primary text should sound welcoming, human, and convey the intention of the feature.
+
+### Secondary text
+This optional text is used to inform the user about the feature in more detail. Secondary text should be brief and non-redundant if possible. From the text, users should  be able to understand the general purpose of the feature and how it may help them accomplish a goal.
+
+### Primary action
+Blankslates can and are encouraged to use one primary action. This button should lead to a feature or component creation flow. Button copy should be keep brief and descriptive. If a button requires further specification, consider adding an Octicon.
+
+### Secondary action
+Secondary actions are optional and are represented by a text link located below the primary action button. A secondary action is used to direct a user to additional content about the feature. This might look like "[Learn more about X](#)" or "[Check out the guide on X](#)" or simply "[Learn more](#)".
+
+### Border
+The border is invisible by default, but can be added to help define the structure of the blankslate component when needed. This can be particularly helpful in page layouts where the blankslate is not the only content on the screen. For implmenetation, check out the [Primer CSS blankslate component](https://primer.style/css/components/blankslate#add-border).
+
+## Variations
+Empty states have multiple variations that can be used in different contexts.
+
+### Pictograms
+[Pictograms](https://ghicons.github.com/) present a great preview of what should be in place of the empty state. Blankslates should default to using a pictogram for graphic elements.
+
+![pictogram blankslate](https://user-images.githubusercontent.com/6846673/62806619-39e39900-baa8-11e9-9de0-5b2f511d63da.png)
+
+### Illustrations
+Illustrations can be used to give more personality to a blankslate and prioritize it as a feature worth learning.
+
+![illustration blankslate](https://user-images.githubusercontent.com/6846673/62806617-394b0280-baa8-11e9-9abc-ccbb76a9f63f.png)
+
+### Compact
+Compact states are a great way to engage and educate a user in places where there are more UI elements and the area is much more condensed. Generally, this would be in places where the content or feature would be left-aligned. While the border is optional for all empty states, the Compact variation requires it to better support page structure.
+
+![compact blankslate](https://user-images.githubusercontent.com/6846673/63041086-dbd5fd80-be7b-11e9-8929-3a9c72f53edc.png)
```

Comment:
I might show the compact blankslate with and without the border or just without the border since that is the default.